pyomeca/analogs.py

```diff
@@ -73,7 +74,30 @@ def from_2d(m):
 
     @classmethod
     def from_mot(cls, filename):
```

Comment:
As a general thought, 
Do you think it would make sense to specify: from_opesim_mot?

Reply:
Not really as both formats were created by opensim devs

Reply:
My thought was more about: what if another company decides to create files that uses the same names. That may be overthinking though lol

Reply:
Yeah that __is__ over engineered haha!